app/main.py

```diff
@@ -0,0 +1,22 @@
+import typing as tp
+import logging
+
+from flask import Flask, jsonify, Response
+
+import settings
+
+logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
```

Comment:
this is fine for now, later it's better to use dict/file config

Reply:
will done in future(this is fine for now)